warehouse/internal/repo/staging.go

```diff
@@ -128,10 +128,15 @@ func (sf *StagingFiles) Insert(ctx context.Context, stagingFile *model.StagingFi
 	if err != nil {
 		return id, fmt.Errorf("marshaling schema: %w", err)
 	}
-
-	bytesPerTableJSON, err := jsonrs.Marshal(stagingFile.BytesPerTable)
```

Comment:
This case wasn't caught by the existing tests since in the DB `bytes_per_table` was being stored as `null` string